pkg/controller/disruption/disruption.go

```diff
@@ -309,10 +309,15 @@ func (dc *DisruptionController) addDb(obj interface{}) {
 }
 
 func (dc *DisruptionController) updateDb(old, cur interface{}) {
-	// TODO(mml) ignore updates where 'old' is equivalent to 'cur'.
```

Comment:
This TODO is about increasing performance by increasing the complexity of our code. I'm not sure it's a good tradeoff if we haven't measured how often this happens?

Reply:
Actually, I was assuming this was about user-instigated changes. If it is also about detecting status changes, then this is not a good idea. We should reduce the number of syncs if we are worried about this. I recommend just deleting this todo instead of implementing it :)

fyi @mml